indico/modules/events/papers/templates/_contributions.html

```diff
@@ -1,10 +1,11 @@
-{% macro render_paper_section(contribution, show_paper=True) %}
+{% macro render_paper_section(contribution, show_paper=true) %}
+    {% set can_access = contribution.is_user_associated(session.user, check_abstract=true) %}
     {% set paper = contribution.paper %}
     {% set show_paper = paper.accepted_revision and show_paper %}
     {% set show_paper_status = paper.last_revision and contribution.can_manage(session.user, 'submit') %}
     {% set show_paper_submission = contribution.event_new.cfp.is_open and contribution.can_manage(session.user, 'submit') %}
 
-    {% if show_paper or show_paper_status or show_paper_submission %}
+    {% if (show_paper or show_paper_status or show_paper_submission) and can_access %}
```

Comment:
This check is wrong, `can_access` should be called `can_submit` and only apply to the paper status/submission - showing the accepted paper revision doesn't need any special access.